modules/1_bastion/bastion.tf

```diff
@@ -213,7 +218,8 @@ resource "null_resource" "bastion_packages" {
     }
     provisioner "remote-exec" {
         inline = [
-            "pip3 install ansible -q"
+            "pip3 install ansible -q",
+            "sudo ppc64_cpu --smt=2"
```

Comment:
smt=2 or 4?

Reply:
Good catch.. this change was not intended.